README.md

```diff
@@ -177,11 +181,19 @@ maintains [cometbft.com](https://cometbft.com).
 [sg-badge]: https://sourcegraph.com/github.com/cometbft/cometbft/-/badge.svg
 [sg-url]: https://sourcegraph.com/github.com/cometbft/cometbft?badge
 [tests-url]: https://github.com/cometbft/cometbft/actions/workflows/tests.yml
+[tests-url-v038x]: https://github.com/cometbft/cometbft/actions/workflows/tests.yml?query=branch%3Av0.38.x
```

Comment:
Unrelated to this change: in this file ([line 152](https://github.com/cometbft/cometbft/blob/3f675584d4147a082050b88032301a482c70352b/README.md?plain=1#L152)) there is a mention of "CosmosBFT".

Reply:
Good catch! Fixed.